router.php

```diff
@@ -6,6 +6,12 @@
  * your local PHP development server. NOT intended for production whatsoever.
  */
 
+if (php_sapi_name() !== 'cli-server') {
```

Comment:
We use `if ( getenv('PHPENV') !== 'production' ) {` in `_templates/alert.php`, maybe that would be better here?

Reply:
I do not prefer directly dealing with `getenv` as changing those values require `sudo` access to the server. Since `php` 5.4, `cli-server` is present as an `sapi` and should be preferred over custom env vars. Though we must not mix 2 ways to accomplish same thing in 1 project, I hereby amended that to use (opposite of) proposed condition.